apps/docs/src/app/documentation/core-helpers/sections-toolbar/sections-toolbar.component.html

```diff
@@ -12,27 +12,34 @@
         </fd-input-group>
     </div>
     <div class="fd-docs-sidebar-content">
-        <nav *ngFor="let section of sections">
-            <ul class="nav">
-                <div class="side-nav__headers">{{ section.header }}</div>
-                <ng-container *ngFor="let item of section.content">
-                    <li *ngIf="item.url === 'home' || (item | docsSearch: search)">
-                        <div class="nav-item-container">
-                            <a
-                                class="nav-item"
-                                [routerLink]="'/' + item.url"
-                                (click)="onItemClick()"
-                                (keypress)="onKeypressHandler($event)"
-                                tabindex="0"
-                                role="button"
-                                routerLinkActive="nav-item--active"
-                            >
-                                {{ item.name }}
+        <fd-side-nav>
+            <div fd-side-nav-main>
+                <ul fd-nested-list [textOnly]="true">
+                    <li fd-nested-list-item *ngFor="let section of sections" [expanded]="true">
+                        <div fd-nested-list-content>
+                            <a fd-nested-list-link>
+                                <span fd-nested-list-title>{{ section.header }}</span>
                             </a>
+                            <button fd-nested-list-expand-icon></button>
                         </div>
+                        <ul fd-nested-list>
+                            <ng-container *ngFor="let item of section.content">
+                                <li fd-nested-list-item *ngIf="item.url === 'home' || (item | docsSearch: search)">
+                                    <a
+                                        fd-nested-list-link
+                                        [routerLink]="'/' + item.url"
+                                        (keypress)="onKeypressHandler($event)"
+                                        tabindex="0"
+                                        role="button"
+                                    >
+                                        <span fd-nested-list-title>{{ item.name }}</span>
+                                    </a>
+                                </li>
```

Comment:
This will accommodate till section and its items, but under item, we will have again subitems, for example as below form controller can have respective components.


**- Section
  - Component**
    - Buttons
    - Form Controllers
         - Input
	     - checkbox

inside items =>li we need to have an arrow with subitems iterator.